build-tools/packages/build-cli/src/codeCoverage/README.md

```diff
@@ -0,0 +1,62 @@
+# Code-Coverage-Analysis-Tool
```

Comment:
Consider putting this in the docs folder, called something like codeCoverageDetails.md.